docs/content/en/docs-dev/operator-manual/piped/configuration-reference.md

```diff
@@ -69,6 +70,15 @@ spec:
 | gitRemote | string | Remote address of the Git repository used to clone Helm charts. | Yes if type is GIT |
 | sshKeyFile | string | The path to the private ssh key file used while cloning Helm charts from above Git repository. | No |
 
+## ChartRegistry
+
+| Field | Type | Description | Required |
+|-|-|-|-|
+| type | string | The registry type. Currently, only OCI is supported. Default is OCI. | No |
+| address | string | The address to the registry. | Yes |
+| username | string | Username used for the registry authentication. | No |
+| password | string | Password used for the registry authentication. | No |
```

Comment:
If `username` or `password` is not specified, there is nothing to do when piped is invoked regarding the Registry login (so it's the same as not specifying `chartRegistries` field).
Should I add something about this?

Reply:
I think it's enough to leave it that way 👍